cmd/operator-sdk/bundle/generate.go

```diff
@@ -0,0 +1,60 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type AnnotationMetadata struct {
+	Annotations AnnotationType `yaml:"annotations"`
+}
+
+type AnnotationType struct {
+	Resources string `yaml:"operators.operatorframework.io.bundle.resources"`
+	MediaType string `yaml:"operators.operatorframework.io.bundle.mediatype"`
+}
+
+// newBundleGenerateCmd returns a command that will generate operator bundle
+// annotations.yaml metadata
+func newBundleGenerateCmd() *cobra.Command {
+	bundleGenerateCmd := &cobra.Command{
+		Use:   "generate",
+		Short: "Generate operator bundle metadata and Dockerfile",
+		Long: `The "operator-sdk bundle generate" command will generate operator
```

Comment:
Why is required a `operator-sdk bundle build` and a `operator-sdk bundle generate` one? 
What are the diff between both?

Reply:
@dinhxuanvu based on the current command descriptions, a user wouldn't necessarily be able to tell the difference between the commands. Can you specify that `generate` only creates the operator metadata without building the image, and that a non-default builder can be used to build the metadata image?